cli/Valet/Site.php

```diff
@@ -190,6 +190,29 @@ public function proxies()
         return $proxies;
     }
 
+    /**
+     * Determine if the provided site is a valid site, whether parked or linked and get the site url.
+     *
+     * @param  string  $directory
+     * @return string|false
+     */
+    public function getSiteUrl($directory)
+    {
+        $tld = $this->config->read()['tld'];
+
+        if ($directory == '.') { // Allow user to use dot as current dir's site `--site=.`
```

Comment:
Should we also allow `./`?
```suggestion
        if ($directory == '.' || $directory == './') { // Allow user to use dot as current dir's site `--site=.`
```